# HG changeset patch
# User Rik <rik@octave.org>
# Date 1451191309 28800
#      Sat Dec 26 20:41:49 2015 -0800
# Node ID 3ec8332e0e01da0d99c1347914dfa7d46d407939
# Parent  fc9cca99b2de6754ed46800618ff59b6103b1794
maint: Use two newlines between endfunction and start of BIST tests.

* im2double.m, imread.m, iscolormap.m, ntsc2rgb.m:
Use two newlines between endfunction and start of BIST tests.

diff --git a/scripts/image/im2double.m b/scripts/image/im2double.m
--- a/scripts/image/im2double.m
+++ b/scripts/image/im2double.m
@@ -81,16 +81,17 @@ function img = im2double (img, im_type)
               "integer floating points, or unsigned integer class"]);
     endif
 
   else
     print_usage ();
   endif
 endfunction
 
+
 %!assert (im2double ([1 2 3]), [1 2 3])
 %!assert (im2double (single ([1 2 3])), [1 2 3])
 %!assert (im2double (uint8 ([0 127 128 255])), [0 127/255 128/255 1])
 %!assert (im2double (uint16 ([0 127 128 65535])), [0 127/65535 128/65535 1])
 %!assert (im2double (int16 ([-32768 -32767 -32766 32767])),
 %!                   [0 1/65535 2/65535 1])
 
 %!assert (im2double (uint8 ([0 1 255]), "indexed"), [1 2 256])
diff --git a/scripts/image/imread.m b/scripts/image/imread.m
--- a/scripts/image/imread.m
+++ b/scripts/image/imread.m
@@ -103,16 +103,17 @@ function [img, varargout] = imread (file
     error ("imread: FILENAME must be a string");
   endif
 
   [img, varargout{1:nargout-1}] = ...
     imageIO ("imread", @__imread__, "read", filename, varargin{:});
 
 endfunction
 
+
 %!testif HAVE_MAGICK
 %! vpng = [ ...
 %!  137,  80,  78,  71,  13,  10,  26,  10,   0,   0, ...
 %!    0,  13,  73,  72,  68,  82,   0,   0,   0,   3, ...
 %!    0,   0,   0,   3,   8,   2,   0,   0,   0, 217, ...
 %!   74,  34, 232,   0,   0,   0,   1, 115,  82,  71, ...
 %!   66,   0, 174, 206,  28, 233,   0,   0,   0,   4, ...
 %!  103,  65,  77,  65,   0,   0, 177, 143,  11, 252, ...
diff --git a/scripts/image/iscolormap.m b/scripts/image/iscolormap.m
--- a/scripts/image/iscolormap.m
+++ b/scripts/image/iscolormap.m
@@ -40,13 +40,14 @@ function retval = iscolormap (cmap)
   endif
 
   retval = (isnumeric (cmap) && isreal (cmap)
             && ndims (cmap) == 2 && columns (cmap) == 3
             && isfloat (cmap));
 
 endfunction
 
+
 %!assert (iscolormap (jet (64)))
 %!assert (iscolormap ({0 1 0}), false)
 %!assert (iscolormap ([0 1i 0]), false)
 %!assert (iscolormap (ones (3,3,3)), false)
 %!assert (iscolormap (ones (3,4)), false)
diff --git a/scripts/image/ntsc2rgb.m b/scripts/image/ntsc2rgb.m
--- a/scripts/image/ntsc2rgb.m
+++ b/scripts/image/ntsc2rgb.m
@@ -79,16 +79,17 @@ function rgb = ntsc2rgb (yiq)
   ## truncating / scaling of double rgb values for Matlab compatibility
   rgb = max (0, rgb);
   idx = any (rgb > 1, 2);
   rgb(idx,:) = rgb(idx,:) ./ max (rgb(idx,:), [], 2);
 
   rgb = colorspace_conversion_revert (rgb, sz, is_im, is_nd);
 endfunction
 
+
 %!shared trans
 %! trans = [ 1.0,      1.0,      1.0;
 %!          0.95617, -0.27269, -1.10374;
 %!          0.62143, -0.64681,  1.70062 ];
 
 ## Test pure R, G, B colors
 %!assert (ntsc2rgb ([.299  .596  .211]), [1 0 0], 1e-5)
 %!assert (ntsc2rgb ([.587 -.274 -.523]), [0 1 0], 1e-5)
